BosTau.py

```diff
@@ -0,0 +1,215 @@
+"""
+Catalog definitions for Bos Taurus
+"""
+import math
+import logging
+
+import msprime
+
+import stdpopsim
+
+logger = logging.getLogger(__name__)
+
+###########################################################
+#
+# Genome definition
+#
+###########################################################
+
+
+# Create a string of chromosome lengths for easy parsing
+# Length data was extracted from Ensembl genome browser Eg. for Chr1 ....https://www.ensembl.org/Bos_taurus/Location/Chromosome?r=1:421347-158534110
+_chromosome_data = """\
+chr1    158534110
+chr2    136231102
+chr3    121005158
+chr4    120000601
+chr5    120089316
+chr6    117806340
+chr7    110682743
+chr8    113319770
+chr9    105454467
+chr10   103308737
+chr11   106982474
+chr12   87216183
+chr13   83472345
+chr14   82403003
+chr15   85007780
+chr16   81013979
+chr17   73167244
+chr18   65820629
+chr19   63449741
+chr20   71974595
+chr21   69862954
+chr22   60773035
+chr23   52498615
+chr24   62317253
+chr25   42350435
+chr26   51992305
+chr27   45612108
+chr28   45940150
+chr29   51098607
+chrX    139009144
+"""
```

Comment:
Sorry for the sudden change but this info should now be in an auto-generated file in this directory named `genome_data.py` this file is autogenerated by querying the ensembl REST API. We have a script for updating chromosome info for all species [here](https://github.com/popsim-consortium/stdpopsim/blob/master/update_ensembl_data.py) but you probably want to tweak it to just add cow. (We really need to add at least a simple CLI to that file to make this easier). Finally, you will need to add an `from . import genome_data` to the top of this file and rename it `__init__.py`. Look at some of the other species for an example if you need them.

Reply:
Hello ndukler,

Reply:
Hello @ndukler , I'm trying to run the script for getting the chromosome info but it crashes, it cannot find module called "maintenance". Moreover, I don't know how to tweak it to get only cattle info. That should generate a file called genome_data.py , is that correct?  I've already made some changes including rename of BosTau.py to __init__.py

Reply:
That's correct. Also I suppose you can run it so it updates everything then discard all changes that are not to bosTau. I'll look into running it and get back to you.